Project.toml

```
@@ -11,34 +11,31 @@ InfrastructureModels = "2030c09a-7f63-5d83-885d-db604e0e9cc0"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+PolyhedralRelaxations = "2e741578-48fa-11ea-2d62-b52c946f73a0"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 XMLDict = "228000da-037f-5747-90a9-8195ccbf91a5"
 ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
-PolyhedralRelaxations = "2e741578-48fa-11ea-2d62-b52c946f73a0"
-
-
 
 [compat]
-HiGHS = "~0.3, ~1"
 Dierckx = "~0.4, ~0.5"
+HiGHS = "~0.3, ~1"
 InfrastructureModels = "~0.6, ~0.7"
 Ipopt = "~0.8, ~0.9, ~1"
 JSON = "~0.18, ~0.19, ~0.20, ~0.21"
-JuMP = "~0.22, ~0.23, 1"
+JuMP = "1.15"
 Juniper = ">= 0.4"
```

```suggestion
Juniper = "0.9.2"
```
Once https://github.com/lanl-ansi/Juniper.jl/pull/266 is tagged